### PR TITLE
Metagenome exporters

### PR DESCRIPTION
**Pull request by slebras**

### CODE REVIEW

File: lib/GenomeFileUtil/GenomeFileUtilImpl.py [hunk 1]
```diff
@@ -173,8 +173,13 @@ def metagenome_to_gff(self, ctx, params):
         # ctx is the context object
         # return variables are: result
         #BEGIN metagenome_to_gff
-        print('metagenome_to_gff -- paramaters = ')
-        pprint(params)
+        # print('metagenome_to_gff -- paramaters = ')
```
Comment (jayrbolton) [inline lib/GenomeFileUtil/GenomeFileUtilImpl.py]: Can you just delete these lines

File: lib/GenomeFileUtil/GenomeFileUtilImpl.py [hunk 2]
```diff
@@ -473,6 +478,78 @@ def export_genome_features_protein_to_fasta(self, ctx, params):
...
+        # export to file (building from KBase Genome Object)
+        result = self.metagenome_to_gff(ctx, {
+            'metagenome_ref': params['input_ref']
+        })[0]
+
+        # get assembly
+        if 'assembly_ref' in info:
+            assembly_ref = info['assembly_ref']
+        else:
+            raise ValueError("no 'assembly' associated with this AnnotatedMetagenomeAssembly "
```
Comment (jayrbolton) [inline lib/GenomeFileUtil/GenomeFileUtilImpl.py]: Can you change to "No Assembly associated with this AnnotatedMetagenomeAssembly...'

File: lib/GenomeFileUtil/GenomeFileUtilImpl.py [hunk 3]
```diff
@@ -473,6 +478,78 @@ def export_genome_features_protein_to_fasta(self, ctx, params):
...
+                             "object. Cannot retrieve fasta file. ")
+        # print(('Assembly reference = ' + assembly_ref))
```
Comment (jayrbolton) [inline lib/GenomeFileUtil/GenomeFileUtilImpl.py]: Delete

**Comment by jayrbolton** [comment]
Aprobado

**Comment by slebras** [comment]
@scanon and @jayrbolton, are we okay with merging this?

**Pull request merged**